packages/expo-modules-core/src/ts-declarations/global.ts

```diff
@@ -91,4 +91,19 @@ declare global {
     };
     [key: string]: any;
   };
+
+  interface FormData {
```

Comment:
CI failures regarding FormData are because our jest setup is not transpiling other packages – so far it was just using JS files from the `build` folder, but now some packages (including expo-modules-core) no longer ship with JS files. Someone from the team is looking into it, so could you revert this change?

Reply:
Got it, updated!